projects/schematics/src/migrations/2_0/constructor-deprecations/data/page-resolvers/product-page-meta.resolver.migration.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  FEATURE_CONFIG_SERVICE,
+  PRODUCT_PAGE_META_RESOLVER,
+  PRODUCT_SERVICE,
+  ROUTING_SERVICE,
+  SPARTACUS_CORE,
+  TRANSLATION_SERVICE,
+} from '../../../../../shared/constants';
+import { ConstructorDeprecation } from '../../../../../shared/utils/file-utils';
+
+export const PRODUCT_PAGE_META_RESOLVER_MIGRATION: ConstructorDeprecation = {
+  class: PRODUCT_PAGE_META_RESOLVER,
+  importPath: SPARTACUS_CORE,
+  deprecatedParams: [
+    {
+      className: ROUTING_SERVICE,
+      importPath: SPARTACUS_CORE,
+    },
+    {
+      className: PRODUCT_SERVICE,
+      importPath: SPARTACUS_CORE,
+    },
+    {
+      className: TRANSLATION_SERVICE,
+      importPath: SPARTACUS_CORE,
+    },
```

Comment:
Feature config service in this case have to be specified in deprecatedParams to work.

Reply:
Added that